pom.xml

```diff
@@ -207,6 +207,22 @@
 			<artifactId>bcprov-jdk15on</artifactId>
 			<version>1.59</version>
 		</dependency>
+		<dependency>
+			<groupId>org.powermock</groupId>
+			<artifactId>powermock-api-mockito2</artifactId>
+			<version>2.0.0-beta.5</version>
+		</dependency>
+		<dependency>
+			<groupId>org.powermock</groupId>
+			<artifactId>powermock-module-junit4</artifactId>
+			<version>2.0.0-beta.5</version>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>com.diffblue</groupId>
+			<artifactId>deeptestutils</artifactId>
+			<version>1.9.0</version>
+		</dependency>

```

Comment:
Are you using the `deeptestutils` dependency? Or could it be removed?

Reply:
Removed